src/main/java/org/broadinstitute/hellbender/utils/codecs/gtf/EnsemblGtfCodec.java

```diff
@@ -120,7 +121,28 @@ void incrementLineNumber() {
 
     @Override
     String getUcscVersionNumber() {
-        return getVersionFromHeader();
+        return version;
+    }
+
+    @Override
+    /**
+     * {@inheritDoc}
+     *
+     * Because ENSEMBL GTF files are strictly a superset of GENCODE GTF files, we need to do some extra checks here to
```

Comment:
Oh, I didn't realize that.  Does that imply that the ensemble codec should extend GencodecodeGtfCodec instead of being a parallel class to it?  Maybe something to think about in the future.

Reply:
Yeah - I didn't realize it until doing the work for this issue.

It's probably worth looking into doing that - it would be nice to have a single point of control for this.  I also need to change the name of some other classes in response to having multiple kinds of GTF files, but I wanted to leave that out of this PR for simplicity.

Reply:
I agree, that's a different pr.